modules/olmv1-operator-api.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-managing-catalogs.adoc
+
+:_content-type: CONCEPT
+
+[id="olmv1-operator-api"]
+== Operator API
+
+Operator Controller provides a new `Operator` API object (`operator.operators.operatorframework.io`), which is a single resource that represents an instance of an installed Operator. The `Operator` API streamlines management of installed Operator by consolidating user-facing APIs into a single object.
+
+[IMPORTANT]
+====
+In {olmv1}, `Operator` objects are cluster-scoped. This differs from earlier OLM versions where Operators could be either namespace-scoped or cluster-scoped, depending on the configuration of their related `Subscription` and `OperatorGroup` objects.
+
+For more information about the earlier behavior, see _Multitenancy and Operator colocation_.
+====
+
+.Example `Operator` object
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: <operator_name> <1>
+spec:
+  packageName: <package_name> <2>
+  channel: <channel_name> <3>
+  version: <version_number> <4>
+----
+<1> Name of the Operator.
+<2> Name of a package in an Operator bundle.
+<3> Installs the latest release published to the specified channel. Updates to the channel are automatically installed.
+<4> Required. Specifies a target version to install.
```

Comment:
My understanding is that if you specify a channel, then the latest version in the channel is installed and updates are automatically applied. 

If you specify a version, then the target version is pinned to that version. If you don't specify either, what is supposed to happen is you get the latest version and it automatically updates. But there is a bug that hasn't been backported to 4.14 yet.

Joe mentioned in the doc read through, that at this point, specifying a version and channel doesn't get you anything. In the future, with version ranges, then you can specify a version range and set a channel.